app/derivative_services/vips_derivative_service.rb

```diff
@@ -98,14 +99,25 @@ def vips_image
     @vips_image ||=
       begin
         image = image_from_file(filename.to_s)
-        if image.height >= REDUCTION_THRESHOLD || image.width >= REDUCTION_THRESHOLD
-          image.resize(0.5)
-        else
-          image
-        end
+        resize(image)
       end
   end
 
+  def resize(image)
+    if image.height >= REDUCTION_THRESHOLD || image.width >= REDUCTION_THRESHOLD
```

Comment:
That's not what this code was about before - it wasn't about making all the images smaller than the given size, it was about cutting them in half if they were above it, because a bunch of testing with scrolls and stuff showed that it got performance above a certain mark while keeping the quality good enough to see the grain.

Reply:
Reverted